src/main/java/no/nav/fo/veilarbregistrering/oppgave/resources/OppgaveApi.java

```diff
@@ -0,0 +1,16 @@
+package no.nav.fo.veilarbregistrering.oppgave.resources;
+
+import io.swagger.annotations.*;
+
+@Api(value = "OppgaveResource")
+public interface OppgaveApi {
```

Comment:
Skiller ut Swagger-doc i egen fil for å gjøre det mer ryddig. Bruker interface for å dokumentere APIet.